### PR TITLE
download tool

### PR DESCRIPTION
**Pull request by jmikedupont2**

### CODE REVIEW

File: load.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (mudler) [inline load.sh:1]:
```suggestion
#!/bin/bash
```

File: README.md [hunk 2]
```diff
@@ -9,4 +9,14 @@ The model gallery is a curated collection of models created by the community and
 
 We encourage contributions to the gallery! However, please note that if you are submitting a pull request (PR), we cannot accept PRs that include URLs to models based on LLaMA or models with licenses that do not allow redistribution. Nevertheless, you can submit a PR with the configuration file without including the downloadable URL.
 
+
+## Helper tool
+
+To load a model from main onto localhost
+
+```shell
+bash  ./load.sh wizard
```
Comment (mudler) [inline README.md:18]:
```suggestion
bash ./load.sh wizard
```

**Review by mudler — APPROVED** [review]
just small nits here and there, thank you!

**Review by lunamidori5 — COMMENTED** [review]
If you have time, id love more info on the goal of this tool!

[1] What does this tool do that the gallery does not already do?
[2] Will there be a ``.bat`` or Windows NT copy of the tool?
[3] What is the full goal of the tool?

I am all for it but just some more info would be lovely!

**Comment by jmikedupont2** [comment]
> If you have time, id love more info on the goal of this tool!
> 
> [1] What does this tool do that the gallery does not already do? 
This is a helper tool to allow you to apply a yaml file quickly from the checked out directory.

[2] Will there be a `.bat` or Windows NT copy of the tool? 
When you find a windows user? I suppose powershell can curl

[3] What is the full goal of the tool?
Less guis more clis
> I am all for it but just some more info would be lovely!

**Review by lunamidori5 — COMMENTED** [review]
> > If you have time, id love more info on the goal of this tool!
> > [1] What does this tool do that the gallery does not already do?
> > This is a helper tool to allow you to apply a yaml file quickly from the checked out directory.
> 
> [2] Will there be a `.bat` or Windows NT copy of the tool? When you find a windows user? I suppose powershell can curl
> 
> [3] What is the full goal of the tool? Less guis more clis
> 
> > I am all for it but just some more info would be lovely!

@jmikedupont2 
Hi Windows user here! wget can be supported on windows with a easy install. If you want to go that route alright @mudler did approve so Ill approve after the changes @mudler requested are taken. Thank you for your time!

**Comment by jmikedupont2** [comment]
here from chatgpt for the windows bat users

Certainly! You can convert the provided Bash script to a Windows Command Prompt (CMD) command using `curl` like this:

```cmd
@echo off
set NAME=%1
curl http://localhost:8080/models/apply -H "Content-Type: application/json" -d "{\"url\": \"https://raw.githubusercontent.com/go-skynet/model-gallery/main/%NAME%.yaml\", \"name\": \"%NAME%\"}"
```

Make sure to save this as a `.bat` file, and you can run it by providing the `NAME` as an argument. For example:

```cmd
script.bat my_model
```

Replace `script.bat` with your chosen filename and `my_model` with the desired `NAME`.

**Review by lunamidori5 — APPROVED** [review]

**Comment by jmikedupont2** [comment]
and powershell Certainly! You can convert the provided Bash script to a PowerShell script like this:

```powershell
$NAME = $args[0]
$body = @{
    url = "https://raw.githubusercontent.com/go-skynet/model-gallery/main/$NAME.yaml"
    name = $NAME
} | ConvertTo-Json

Invoke-RestMethod -Uri "http://localhost:8080/models/apply" -Method Post -Headers @{"Content-Type"="application/json"} -Body $body
```

Save this as a `.ps1` file and run it in PowerShell. You can run it by providing the `NAME` as an argument. For example:

```powershell
.\script.ps1 my_model
```

Replace `script.ps1` with your chosen filename and `my_model` with the desired `NAME`.

**Pull request merged**